src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/AddSingleCaseToMultipleService.java

```diff
@@ -92,13 +90,25 @@ public void addSingleCaseToMultipleLogic(String userToken, CaseData caseData, St
 
     }
 
+    private String createMultipleReferenceLink(CaseData caseData)
+    {
+        var newMultipleReference = caseData.getMultipleReference() != null ?
+                caseData.getMultipleReference().toString() : "";
+        var ethosReference = caseData.getEthosCaseReference() != null ?
+                caseData.getEthosCaseReference().toString() : "";
+        var url =  "<a href=\"../cases/case-details/"+ethosReference+"\">"+
```

Comment:
You should be consistent with your formatting. Have a space around the +

Reply:
done